addons/mail/static/src/core/common/message_in_reply.xml

```diff
@@ -17,9 +17,12 @@
                                     <t t-out="props.message.parent_id.inlineBody"/>
                                     <em t-if="props.message.parent_id.edited" class="smaller fw-bold text-500"> (edited)</em>
                                 </t>
-                                <t t-elif="props.message.parent_id.attachment_ids.length > 0">
-                                    <span class="me-2 fst-italic">Click to see the attachments</span>
-                                    <i class="fa fa-image"/>
+                                <t t-else="">
+                                    <span class="fst-italic">Click to see the attachments</span>
+                                </t>
+                                <t t-if="props.message.parent_id.hasAttachments">
+                                    <span class="ms-2"/>
+                                    <i class="position-absolute top-0 end-0 fa fa-image" />
```

Comment:
Messaging menu is showing icon prefix when message has attachment:
<img width="478" height="422" alt="Image" src="https://github.com/user-attachments/assets/e69f96fa-bc9d-480b-a89f-44bbf87171aa" />

I think we could do the same: show icon in the prefix:
<img width="225" height="82" alt="Image" src="https://github.com/user-attachments/assets/17a0c7f0-25c4-4905-a382-a7e1dbf3a171" />

Reply:
I don't like that it's the inverse of how messages are ordered (body/attachment, but preview is attachment/body).
But it solves problems and is easier to manage and implement.